### ISSUE TITLE
Native deleteEntity / deleteVehicle doesnt work

### ISSUE THREAD

**Issue by YannBCF** [altmp/altv-issues#1276]
### Description of the problem

The natives deleteEntity and deleteVehicle doesnt work for serverside and clientside created vehicles.
deleteEntity however works for peds and objects
deleteObject works for objects
deletePed works for peds

## clientside approach
client.js
```js
const vehModel = alt.hash('stockade3');
const { x, y, z } = alt.Player.local.pos;

game.requestModel(vehModel);
setTimeout(() => {
    const veh = game.createVehicle(vehModel, x, y, z, 0, false, false, false);
    console.log('Veh handler', veh);
    setTimeout(() => {
        game.deleteEntity(veh);
        // game.deleteVehicle(veh);
    }, 2500);
}, 2500);
```

## serverside approach
client.js
```js
alt.onServer('test', veh => {
    setTimeout(() => {
        if (veh.valid) {
            console.log('Vehicle destroying..')
            game.deleteVehicle(veh);
        }
    }, 1000);
});
```
server.js
```js
const { x, y, z } = player.pos;
const veh = new alt.Vehicle('italirsx', player.pos, player.rot);
alt.emitClient(player, 'test', veh);
```

### Expected behaviour

The car should be deleted

### Operating system

Windows 10

### Version

release/8.4

**Comment by C0kkie** [comment]
Will be fixed in release/8.4

**Comment by xtance** [comment]
On server with altv-release 8.4 this bug persists.
Since this bug is clientside, does it mean it would be fixed automatically with altv update?

**Comment by FabianTerhorst** [comment]
the fix wasnt included in 8.4. its inside 8.5 and yes its clientside only.
